globals.py

```diff
@@ -4,6 +4,7 @@
 sell_ready = threading.Event()
 stop_threads = False
 old_coins = {}
+latest_listing = "BTC"
```

Comment:
I vote initializing this variable with "" instead of "BTC". I know the buy won't start until the event has been signaled by the announcement thread, but I still feel it's better to be safe than sorry.